classes/system/openstack/common/mk20_stacklight_basic.yml

```diff
@@ -33,11 +33,12 @@ parameters:
           address: 172.16.10.107
           names:
           - mon
+          - nagios
```

Comment:
We don't do it for elasticsearch and influxdb, so I would not make nagios a special case.